autopilot/contractor.go

```diff
@@ -140,28 +145,39 @@ func (c *contractor) performContractMaintenance(cfg Config) error {
 		return err
 	}
 
-	// fetch all active contracts
-	active, err := c.ap.bus.Contracts()
+	// fetch all active contracts and their latest revisions.
+	contractData, err := c.ap.bus.Contracts()
```

Comment:
is it crazy to ask the worker for contracts? "zipping" contracts and revisions is terrible

Reply:
a `worker.Contract` type makes more sense now that revisions are fetched on the fly